Runtime/Scripts/RTPParameters.cs

```diff
@@ -202,6 +218,26 @@ internal RTCRtpCapabilities(RTCRtpCapabilitiesInternal capabilities)
         }
     }
 
+    /// <summary>
+    /// 
+    /// </summary>
+    public class RTCRtpTransceiverInit
+    {
+        public RTCRtpTransceiverDirection direction;
+        public RTCRtpEncodingParameters[] sendEncodings;
+        public MediaStream[] streams;
+
+        internal RTCRtpTransceiverInitInternal Cast()
+        {
+            return new RTCRtpTransceiverInitInternal
+            {
+                direction = this.direction,
+                sendEncodings = this.sendEncodings.Select(_ => _.Cast()).ToArray(),
+                streams = streams.Select(_ => _.self).ToArray(),
```

Comment:
throw exception if not set `sendEncodings` or `streams`, these parameter need to accept null.(optional parameter)

```
System.ArgumentNullException : Value cannot be null.
Parameter name: source
---
at System.Linq.Enumerable.Select[TSource,TResult] (System.Collections.Generic.IEnumerable`1[T] source, System.Func`2[T,TResult] selector) [0x0000d] in <351e49e2a5bf4fd6beabb458ce2255f3>:0 
  at Unity.WebRTC.RTCRtpTransceiverInit.Cast () [0x00001] in com.unity.webrtc\Runtime\Scripts\RTPParameters.cs:232 
```